gitgraph

```diff
@@ -237,7 +237,7 @@ function create_dot_graph()
 
 function main()
 {
-    if ! check_dependencies dot dia eog git sed cut
```

Comment:
I'm in a hurry to analyzing code but all is looking like you have ignored my commits like if were your changes. :-1: 

Hint: cherry pick, or merge to change it after.